vendor/bat-native-ads/include/bat/ads/ads_client.h

```diff
@@ -133,7 +133,8 @@ class ADS_EXPORT AdsClient {
 
   // Log |training_instance|.
   virtual void LogTrainingInstance(
-      const brave_federated::mojom::TrainingInstancePtr training_instance) = 0;
+      const std::vector<brave_federated::mojom::CovariatePtr>
```

Comment:
this is an array of Covariate, so we should use covariates instead of training_instance as the training instance is created later